### PR TITLE
Add dragdropfiles event

### PR DESCRIPTION
**Pull request by bcomnes**
Dispatch dragdropfiles with FileSystemDirectoryHandle or FileSystemFileHandle entires in a handles array on the event.

### CODE REVIEW

File: api/internal/init.js [hunk 1]
```diff
@@ -69,6 +74,24 @@ if ((globalThis.window || globalThis.self) === globalThis) {
   }
 }
 
+// webview only features
+if ((globalThis.window) === globalThis) {
+  globalThis.addEventListener('dragdropfiles', async (event) => {
+    const { files } = event.detail
+    const handles = []
+    for (const file of files) {
```
Comment (jwerle) [inline api/internal/init.js:82]: Can we add an Array.isArray check 🙏🏼

File: api/internal/init.js [hunk 2]
```diff
@@ -69,6 +74,24 @@ if ((globalThis.window || globalThis.self) === globalThis) {
...
+      const stats = await fs.stat(file)
```
Comment (jwerle) [inline api/internal/init.js:83]: Can we add a `typeof file === string` check?

File: api/internal/init.js (hunk 1 above)
Comment (bcomnes) [reply api/internal/init.js]: Done

File: api/internal/init.js (hunk 2 above)
Comment (bcomnes) [reply api/internal/init.js]: Done

**Comment by bcomnes** [comment]
@jwerle I think that covers your requests?

**Review by jwerle — APPROVED** [review]

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**